src/Api/Application.php

```diff
@@ -97,13 +134,18 @@ protected function getDefaultInputDefinition(): InputDefinition
     protected function getDefaultCommands(): array
     {
         $commands = parent::getDefaultCommands();
+        $commands[] = new VersionCommand($this);
```

Comment:
Why is this not added dynamically?

Reply:
What do you mean by dynamically? This is the basic command that always must be there to determine what the API is capable of.

Reply:
Well, not quite. The API version will never change again, so …

Reply:
So? The command is still relevant for everything to get features etc. 
Also, the version _might_ change if we come up with something new, like UI for manager configuration.